src/DEM/SolidBoundary/RectangularPlaneSolidBoundary.cc

```diff
@@ -37,9 +37,9 @@ distance(const Vector& position) const {
   const double signX = (q.x() > 0.0 ? 1. : -1.);
```

Comment:
Minor quibble, but these recreate the sgn() functions we provide in Utilities/SpheralFunctions.hh (I know, a very generic/non-descriptive file name).

Reply:
ah gotcha yeah no point in reinventing the wheel, I'll swap those out.

Reply:
On second look, I realized I was just implementing bounds on that Vector's values in a confusing way

Reply:
Even better, looks good to me.